doc/model_output.md

```diff
@@ -0,0 +1,37 @@
+## Model Output Type
+
+There are two broad kinds of output types of power models provided by the model server: absolute power model (Abs) and dynamic power model (Dyn). 
+
+**Absolute Power Model (Abs):** power model trained by measured power (including the idle power)
+
+**Dynamic Power Model (Dyn):** power model trained by delta power (measured power with workload - idle power)
+
+The absolute power model is supposed to use for estimating node-level power while the dynamic power model is supposed to use for estimating pod-level power. 
+
+Note that, the node-level power would be applied in the case that the measured power or low-level highly-dependent features (such as temperature sensors) are not available such as in virtualized systems, non-instrumented systems.
+
+For each kind, the model can be in four following forms.
+
+Key Name| Model Form|Output per single input|
+---|---|---|
+Power|achived model|single power value|
+ModelWeight|weights for normalized linear regression model|single power value|
+ComponentPower|achived model|multiple power values for each components (e.g., core, dram)
+ComponentModelWeight|weights for normalized linear regression model|multiple power values for each components (e.g., core, dram)|
+
+The achived model allows more choices of learning approachs to estimate the power which can expect higher accuracy and overfitting resolutions while weights of linear regression model provides simplicity and lightweight. **Accordingly, the achived model is suitable for the systems that run intensive and non-deterministic workloads which require accurate prediction while the weights of linear regression is suitable for the systems which run for deterministic workloads in general.**
```

Comment:
Do you mean archived? What is achived model?

Reply:
Yes. I will put the link to the format page.